tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -53,18 +48,20 @@ def test_snapshot_current_version(uvm_nano):
     vm = uvm_nano
     vm.start()
 
-    version = get_firecracker_version_from_toml()
-    # normalize to a snapshot version
-    version = f"{version.major}.{version.minor}.0"
     snapshot = vm.snapshot_full()
 
     # Fetch Firecracker binary for the latest version
     fc_binary = uvm_nano.fc_binary_path
+    # Get supported snapshot version from Firecracker binary
+    snapshot_version = (
+        run_cmd(f"{fc_binary} --snapshot-version").stdout.strip().splitlines()[0]
+    )
```

Comment:
nit: this could be a method of Microvm class. But since it's not called often it may not be worth it.